BitwardenShared/UI/Auth/VaultUnlockSetup/VaultUnlockSetupProcessor.swift

```diff
@@ -5,17 +5,19 @@
 class VaultUnlockSetupProcessor: StateProcessor<VaultUnlockSetupState, VaultUnlockSetupAction, VaultUnlockSetupEffect> {
     // MARK: Types
 
-    typealias Services = HasAuthRepository
+    typealias Services = DefaultVaultUnlockSetupHelper.Services
         & HasBiometricsRepository
```

Comment:
can we nuke `HasBiometricsRepository` since its within `DefaultVaultUnlockSetupHelper.Services`?